front/style/pages/_citizen-projects.scss

```diff
@@ -304,6 +304,10 @@
             }
         }
     }
+
+    .committee-follow--anonymous__link {
```

Comment:
What do you thin of putting it like this : 
```
.committee-follow {
  $--anonymous__link {
 }
}
```

Reply:
Yes good idea but is it really necessary because there is just one css modifition for this class? Maybe to be consistent ?

Reply:
Just to be consistent